### PR TITLE
Register fixes

### PR DESCRIPTION
**Pull request by ilzeLourens**

### CODE REVIEW

File: skietbaan_fe/src/components/Login.js [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
+  Button,
+  Form
 } from 'reactstrap';
 import '../components/RegisterStyles.css';
-import {  validateUsername } from './Validators.js';
+import { validateUsername } from './Validators.js';
 import { getCookie } from './cookie.js';
-import {URL} from '../actions/types.js';
-import header from'../components/assets/header.png';
+import { BASE_URL, URL } from '../actions/types.js';
```
Comment (FredJEhlers) [inline skietbaan_fe/src/components/Login.js]: Each on its own line please 
```
import {
  BASE_URL,
  URL
}
```

File: skietbaan_fe/src/components/Login.js [hunk 2]
```diff
@@ -29,6 +31,27 @@ class Login extends Component {
...
+      headers: {
+        'Accept': 'application/json',
+        'Content-Type': 'application/json'
+      }
+    })
+      .then(response => response.json())
+      .then(data => this.setState({
+        users: data,
+      })).then(function (data) {
+        console.log(data);
```
Comment (FredJEhlers) [inline skietbaan_fe/src/components/Login.js]: Remove debug logs

File: skietbaan_fe/src/components/Login.js [hunk 3]
```diff
@@ -46,70 +69,84 @@ class Login extends Component {
...
         body: JSON.stringify(RequestObject)
-      }).then(function(response) {
-        return response.json();
-      }).then( function(data) {
-        if(typeof data === "object")
-        {
-          document.cookie = "token =" +data.token+"; expires =Wed, 18 Dec 2030 12:00:00 UTC";
+      }).then(response => response.json()).then(data => {
+        if (typeof data === "object") {
+          document.cookie = "token =" + data.token + "; expires =Wed, 18 Dec 2030 12:00:00 UTC";
```
Comment (FredJEhlers) [inline skietbaan_fe/src/components/Login.js]: Why is this cookie's lifespan so large 😂

File: skietbaan_fe/src/components/RegisterStyles.css [hunk 4]
```diff
@@ -216,7 +216,7 @@
 
 .red-background{
   background-color: #EA241A;
-  border-radius: 13px;
+  border-radius: 18px
```
Comment (FredJEhlers) [inline skietbaan_fe/src/components/RegisterStyles.css]: missing `;`

File: skietbaan_fe/src/components/Login.js (hunk 1 above)
Comment (ilzeLourens) [reply skietbaan_fe/src/components/Login.js]: Im not using Base url I'll just remove it

File: skietbaan_fe/src/components/Login.js (hunk 3 above)
Comment (ilzeLourens) [reply skietbaan_fe/src/components/Login.js]: Once of login of sort !

File: skietbaan_fe/src/components/Login.js (hunk 2 above)
Comment (ilzeLourens) [reply skietbaan_fe/src/components/Login.js]: Removed

File: skietbaan_fe/src/components/RegisterStyles.css (hunk 4 above)
Comment (ilzeLourens) [reply skietbaan_fe/src/components/RegisterStyles.css]: Added

**Pull request merged**